docs/src/further_topics/ugrid/operations.rst

```diff
@@ -566,8 +566,8 @@ Here's another example using a global cubed-sphere data set:
             Auxiliary coordinates:
                 time                            x       -
             Cell methods:
-                mean                        time (300 s)
-                mean                        time_counter
+                0                           time: mean: (interval: 300 s)
+                1                           time_counter: mean
```

Comment:
I don't want to totally upset the apple-cart, but I don't really see the value of having the index numbers here -- even though I believe it **_is_** true that order is significant, 
e.g. you can have "time: mean over years time: max within years" 
(and the reverse order means something _different_).

To me, it would make sense for the index column to list the coord (or equivalent measure), i.e. the bit before the colon.  Something like this:
```
    Cell methods:
        time:             mean (interval: 300 s)
        time_counter:     mean
```

And now, looking back to the original proposal in #4729, I see that is what was originally proposed.
What do you think about this, @trexfeathers ?